vector/v.surf.idw/read_sites.c

```diff
@@ -86,30 +86,31 @@ void read_sites(const char *name, const char *field_name, const char *col,
             continue;
 
         if (!with_z) {
-            int cat, ival, ret;
+            int cat;
 
             /* TODO: what to do with multiple cats */
             Vect_cat_get(Cats, field, &cat);
             if (cat < 0)        /* skip features without category */
                 continue;
 
             if (col) {
```

Comment:
This is one of clang warnings with `-Wall` I have in store to address, but not yet come to.
Clang gives the `-Wsometimes-uninitialized` warning: "'ret' is used uninitialized whenever 'if' condition is false".

Reply:
Should add: with this PR that warning will be fixed. 👍